aviatrix/resource_aviatrix_transit_gateway.go

```diff
@@ -1166,9 +1166,8 @@ func resourceAviatrixTransitGatewayCreate(d *schema.ResourceData, meta interface
 			return fmt.Errorf("'enable_firenet' is not supported in AWSChina (1024) or AzureChina (2048)")
 		}
 		if enableTransitFireNet {
-			// Transit FireNet function is not supported for Azure China
-			if !goaviatrix.IsCloudType(cloudType, goaviatrix.AWSRelatedCloudTypes|goaviatrix.GCPRelatedCloudTypes|goaviatrix.AzureArmRelatedCloudTypes^goaviatrix.AzureChina|goaviatrix.OCIRelatedCloudTypes) {
-				return fmt.Errorf("'enable_transit_firenet' is only supported in AWS (1), GCP (4), Azure (8), OCI (16), AzureGov (32), AWSGov (256), AWS China (1024), AWS Top Secret (16384) and AWS Secret (32768)")
+			if !goaviatrix.IsCloudType(cloudType, goaviatrix.AWSRelatedCloudTypes|goaviatrix.GCPRelatedCloudTypes|goaviatrix.AzureArmRelatedCloudTypes|goaviatrix.OCIRelatedCloudTypes) {
+				return fmt.Errorf("'enable_transit_firenet' is only supported in AWS (1), GCP (4), Azure (8), OCI (16), AzureGov (32), AWSGov (256), AWS China (1024), Azure China (2048), AWS Top Secret (16384) and AWS Secret (32768)")
```

Comment:
since you are touching this could you please remove the comment about AWS Top Secret/AWS Secret from the log message? We don't support those anymore. (also below)

Reply:
done